src/Row.tsx

```diff
@@ -47,6 +48,7 @@ function Row<R, SR = unknown>({
       onDragEnter={wrapEvent(handleDragEnter, onDragEnter)}
       onDragOver={wrapEvent(handleDragOver, onDragOver)}
       onDrop={wrapEvent(preventDefault, onDrop)}
+      style={{ top }}
```

Comment:
Just 1 concern here. Let's say if the user has their own `RowRenderer` and they forgot to add this `top` to their wrapper. The whole layout would be messed up, right?

Reply:
Yuyp. If you pass `top` back to `Row`, or to your wrapper, you'll be fine though.

Reply:
To be honest, I don't know if this worth it given by 5ms to 3ms improvement. As it makes the `RowRenderer` error prone. And I bet people will create issues for this case when they use `CustomRowRenderer`.

Reply:
Aren't custom row renderers already error-prone?